lua/sim/Projectile.lua

```diff
@@ -571,9 +563,8 @@ Projectile = ClassProjectile(ProjectileMethods, DebugProjectileComponent) {
     ---@param self Projectile
     OnLostTarget = function(self)
         local bp = self.Blueprint.Physics
-        local trackTarget = bp.TrackTarget
-        local trackTargetGround = bp.TrackTargetGround
-        if trackTarget and (not trackTargetGround) then
+        if not bp.TrackTarget then return end
+        if not bp.TrackTargetGround then
```

Comment:
```suggestion
        if bp.TrackTarget and not bp.TrackTargetGround then
```
This gives smaller bytecode with identical behavior (https://luac.nl/s/1677ff302e972403375eb51bde1), and it is easier to read.